ghmirror/decorators/checks.py

```diff
@@ -62,7 +63,10 @@ def wrapper(*args, **kwargs):
 
         # Using the Authorization header to get the user information
         user_url = f"{GH_API}/user"
-        resp = conditional_request(method="GET", url=user_url, auth=authorization)
+        # Create a new session is fine here, cause it's not called often
+        resp = conditional_request(
+            session=requests.Session(), method="GET", url=user_url, auth=authorization
+        )
```

Comment:
but this session won't be closed

Reply:
I'd assume garbage collection to take care of this. Can add a `with` context around it to be sure

Reply:
Cause circular import dependencies, and it doesn't really matter, cause it should only be one request

Reply:
strange, it lives in its own module, what circular import we have? The number of session object depends on how many different `Authorization` sent to server, so still can be many.

Reply:
oh, `__init__.py` inside `app` have logic, maybe worth putting it in `/ghmirror/extensions/session.py`

Reply:
Yeah, perhaps worth refactoring/cleaning up